### DISCUSSION TITLE
Wordpress 6.2 Compability

### DISCUSSION THREAD

**Discussion by kgarrett67** [WordPress/gutenberg#50350] [Q&A] [answered]
The latest version of Gutenberg (15.7.0) has not been tested with WordPress 6.2. I am hesitant to update the plugin due to this reporting. When will Gutenberg be tested with WordPress 6.2?

![gutenberg_1570](https://user-images.githubusercontent.com/55056055/236296078-b0ef5f0b-77b1-40e0-806c-24d399903adf.png)

**Comment by talldan** [comment, marked as answer]
Oh, thanks for pointing that out, I think it's an oversight. It should say 6.2. 

I'm updating it in #50360. It should be updated in the next Gutenberg release.
